### ISSUE TITLE
Does this play nicely with ASP.NET Core's error pages?

### ISSUE THREAD

**Issue by JosephWoodward** [JosephWoodward/GlobalExceptionHandlerDotNet#2]
Title says it all, look to see how nicely this places with the standard error pages to ensure it's not swallowing them (if possible). The same goes for MVC when support is added.

**Comment by kinetiq** [comment]
At first I thought the project might be dead, and then I realized you are asking yourself a question. What is going on here? :)

**Comment by JosephWoodward** [comment]
Yes, it was more of a reminder to myself, it's certainly not dead though :)
